lib/galaxy/authnz/psa_authnz.py

```diff
@@ -100,6 +101,12 @@ def __init__(self, provider, oidc_config, oidc_backend_config):
 
         if provider in BACKENDS_NAME:
             self._setup_idp(oidc_backend_config)
+            if provider.lower() == 'elixir':
+                # limit default scope to openid and email only
+                from social_core.backends.elixir import ElixirOpenIdConnect
+                ElixirOpenIdConnect.DEFAULT_SCOPE = ['openid', 'email']
+                # remove at_hash validation requirements since it's not provided by AAI for now and is optional according to oidc spec
+                ElixirOpenIdConnect.JWT_DECODE_OPTIONS['verify_at_hash'] = False
```

Comment:
It seems these two configurations are not Galaxy-specific, and are required to setup the Elixir backend anyway. Therefore, IMO a better place for settings these configurations is in [the PSA backend.](https://github.com/python-social-auth/social-core/blob/000f8e5c8b505f2215e25deebf670b708a0578f4/social_core/backends/elixir.py#L9-L18)

Reply:
I agree with your suggestion. I have submitted a pull request with these two changes to PSA. Hope they will make them part of the default in their next release.

https://github.com/python-social-auth/social-core/pull/334

Reply:
Excellent! Thank you.